.github/workflows/release.yml

```diff
@@ -0,0 +1,82 @@
+name: Cli Release
+on:
+  workflow_dispatch:
+    inputs:
+      change-log:
+        description: 'Open API Changelog, Please provide as a string.'
+        required: true
+      version-type:
+        description: 'Version to upgrade, Major: 0, Minor:1, Patch: 2'
+        required: true
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [10.x]
+    steps:
+      - name: Checkout to cli
+        uses: actions/checkout@v2
+      - name: NPM Install
+        run: npm install
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v2
+        with:
+          node-version: ${{ matrix.node-version }}
+          cache: 'npm'
+      - name: Run tests
+        run: |
+          npm ci
+          npm test
+  update-api-definitions-changelog:
+    runs-on: ubuntu-latest
+    needs: [test]
+    steps:
+      - name: Checkout to cli
+        uses: actions/checkout@v2
+      - name: Update Changelog
+        run: |
+          bash scripts/updateApiDefinitions.sh "${{ github.event.inputs.changeLog }}" ${{ github.event.inputs.versionType }}
+  release:
+    runs-on: ubuntu-latest
+    needs: [update-api-definitions-changelog]
+    steps:
+      - name: Checkout to cli
+        uses: actions/checkout@v2
+      - name: Run git and npm update
+        run: |
+          git pull
```

Comment:
[Q] How do we support side branches with github actions?

Reply:
When we run workflow, we can select the branch
![image](https://user-images.githubusercontent.com/83863595/131494348-1e488cc8-8a69-45c0-99e3-d1d7121fb3bf.png)